### PR TITLE
Tag CUDNN v0.3.0

### PR DESCRIPTION
**Pull request by denizyuret**
This version is compatible with cuDNN3 (7.0.64), and cuDNN4 release candidate (4.0.4).

### CODE REVIEW

File: CUDNN/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Compat
+CUDArt
```
Comment (tkelman) [inline CUDNN/versions/0.3.0/requires]: do you still need to support prereleases?

Comment (denizyuret) [inline CUDNN/versions/0.3.0/requires]: What do you suggest?

Comment (tkelman) [inline CUDNN/versions/0.3.0/requires]: that was meant to be on the `julia 0.4-` line below

`julia 0.4-` means "prereleases of julia 0.4 or newer" which includes the first commit after release-0.3 branched off. If you only need to support release versions of julia, `julia 0.4` means "releases of julia 0.4 or newer"

Comment (denizyuret) [inline CUDNN/versions/0.3.0/requires]: We don't necessarily "need to" but the package does support the prerelease
versions, so I think it is fine the way it is.  I will replace with julia
0.4 next version.

**Pull request merged**